### PR TITLE
add change_git_ref method in Repository

### PR DESCRIPTION
**Pull request by RS1980**
The method is defined in the github REST API but missing here

### CODE REVIEW

File: github/Repository.py [hunk 1]
```diff
@@ -966,6 +966,25 @@ def create_git_ref(self, ref, sha):
...
+    def change_git_ref(self, ref, sha, force=False):
+        """
+        :calls: `PATCH /repos/:owner/:repo/git/refs/:ref <http://developer.github.com/v3/git/refs>`_
+        :param ref: string
+        :param sha: string
+        :param force: bool
+        :rtype: :class:`github.GitRef.GitRef`
+        """
+        assert isinstance(ref, str), ref
+        assert isinstance(sha, str), sha
```
Comment (s-t-e-v-e-n-k) [inline github/Repository.py:978]: Missing an assert for force.

File: github/Repository.py [hunk 2]
```diff
@@ -966,6 +966,25 @@ def create_git_ref(self, ref, sha):
         )
         return github.GitRef.GitRef(self._requester, headers, data, completed=True)
 
+    def change_git_ref(self, ref, sha, force=False):
```
[the remark this answers is not in the document]
Comment (RS1980) [reply github/Repository.py:969]: What about update? Would match the github api naming
https://developer.github.com/v3/git/refs/#update-a-reference

**Comment by RS1980** [comment]
Thanx for the review, s-t-e-v-e-n-k
While trying to produce a working unittest, I browsed through PyGithub's GitRef.py module and found the edit method which pretty much does what I try to achieve here.
Hence, I think we don't need a duplicate function for this and my change can be abandoned (my very first public github contribution, how sad ... ;-)

**Comment by s-t-e-v-e-n-k** [comment]
@RS1980 Well spotted -- it's worth grepping for the end point from the GitHub API docs, we tend to be pretty good about putting that in the docstring so users can match them up. But lovely work anyway, thanks for trying!

**Pull request closed without merging**